docs/okeps/okep-5094-layer2-transit-router.md

```
@@ -0,0 +1,980 @@
+and still have 12M keys left.
+
+The easiest way for now is to use option 1. We can always introduce the option 3 later, preserving already-assigned keys.
+We should add error handling in case we run out of tunnel keys both for layer2 transit routers and layer3 transit switches.
+
+#### NAT configuration
+As much as possible everything related to conntrack should not be modified since doing so can affect tcp connections.
+
+The only nat rule that needs to be moved from GR to transit_router is the one SNATing the traffic from the network subnet that goes to the mgmt port. This is needed because now ovn_custer_router is the router directly connected to the layer2 switch (instead of the GR). The rest of the NAT configurations can stay unchanged on the GR, not also that
+there are plans to move the management port SNAT from OVN to iptables, so there were no issue here.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)